src/libraries/Hosting/AspNetCore/CloudAdapter.cs

```diff
@@ -46,6 +48,7 @@ public CloudAdapter(
         {
             _activityTaskQueue = activityTaskQueue ?? throw new ArgumentNullException(nameof(activityTaskQueue));
             _async = async;
+            _logger = logger ?? NullLogger.Instance;
```

Comment:
This line fixes a bug I'm investigating, so I think this change belongs to another PR